src/apps/AppCenter/DiscoverApps/DiscoverApps.js

```diff
@@ -1,121 +1,202 @@
+          <Tag mode="indicator">{status}</Tag>
+        </TagWrapper>
+        <Description theme={theme} compactMode={compactMode}>
+          {description}
+        </Description>
+      </Main>
+    </Card>
+  )
+}
 
-const Main = styled(Card).attrs({ width: '100%', height: '288px' })`
+const Main = styled.section`
   ${unselectable};
   position: relative;
   overflow: hidden;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  padding-top: 25px;
+  height: 100%;
+  width: 100%;
+  white-space: initial;
+
+  ${({ compactMode }) =>
+    compactMode
+      ? `
+          display: grid;
+          grid-template-columns: auto 1fr auto;
+          grid-template-rows: auto auto auto;
+          grid-template-areas:
+            "empty topright"
+            "icon title"
+            "icon description";
+          padding: ${1.5 * GU}px ${1.5 * GU}px ${4 * GU}px ${3 * GU}px;
+        `
+      : `
+          display: flex;
+          flex-direction: column;
+          align-items: center;
+          padding-top: ${5 * GU}px;
+        `}
 `
 
 const Icon = styled.div`
-  height: 64px;
-  margin-bottom: 5px;
-  img {
-    display: block;
-  }
+  height: ${9 * GU}px;
+  width: ${9 * GU}px;
+  ${({ compactMode }) =>
+    compactMode
+      ? `
+        grid-area: icon;
+        margin-right: ${1.5 * GU}px;
+      `
+      : `
+        margin-bottom: ${2 * GU}px;
+      `}
 `
 
 const Name = styled.p`
   display: flex;
   width: 100%;
-  justify-content: center;
-  margin-bottom: 10px;
+  ${textStyle('title4')}
+  ${({ compactMode }) =>
+    compactMode
+      ? `
+        grid-area: title;
+        align-self: flex-end;
+      `
+      : `
+        justify-content: center;
+        margin-bottom: ${1 * GU}px;
+      `}
 `
 
 const TagWrapper = styled.div`
-  max-width: 100%;
-  padding: 0 20px;
-  margin-bottom: 10px;
+  ${({ compactMode, link }) =>
+    compactMode
+      ? `
+        grid-area: topright;
+        text-align: right;
+        position: absolute;
+        right: ${link ? 3.5 * GU : 0}px;
+      `
+      : `
+        max-width: 100%;
+        padding: 0 ${2.5 * GU}px;
+        margin-bottom: ${1 * GU}px;
+      `}
 `
 
-const Description = styled(Text)`
-  padding: 0 1rem;
-  margin-bottom: 30px;
-  text-align: center;
+const Description = styled.p`
+  color: ${({ theme }) => theme.contentSecondary};
+  ${textStyle('body2')};
+  text-align: left;
+
+  ${({ compactMode }) =>
+    compactMode
+      ? `
+        grid-area: description;
+        display: -webkit-box;
+        -webkit-box-orient: vertical;
+        -webkit-line-clamp: 2;
+        overflow: hidden;
+        height: fit-content;
+      `
+      : `
+        flex: 1;
+        text-align: center;
+        padding: 0 1rem;
+      `}
 `
 
-const Action = styled(SafeLink)`
-  position: absolute;
-  bottom: 0;
-  width: 100%;
-  padding-bottom: 30px;
-  text-align: center;
-  text-decoration: none;
+const StyledIconExternal = styled(IconExternal)`
```

Comment:
This icon seems to move around a bit between cards:

<img width="564" alt="Screen Shot 2019-08-26 at 7 20 15 PM" src="https://user-images.githubusercontent.com/4166642/63709137-97147400-c836-11e9-93a5-f87474aa338a.png">

Perhaps we should lift `StyledIconExternal` from `Main` up to the `Button` (since `Button` already has relative positioning applied).

Reply:
This is happening because of the change in `Card` to become a `button` when the `onClick` handler is set (cc @bpierre). I had already fixed this for the regular block `Card`s. The way to fix now is to include `<Card css='flex-direction: column;'` {/*...*/} />` which I have done and will push shortly.
Maybe that should be the default direction for the container buttons Pierre?